Localization/Resources/es-ES/winget.resw

```diff
@@ -1268,4 +1275,78 @@ Especifique uno de ellos con la opción '--source' para continuar.</value>
   <data name="SystemArchitecture" xml:space="preserve">
     <value>Arquitectura del sistema</value>
   </data>
+  <data name="PreserveArgumentDescription" xml:space="preserve">
+    <value>Conserva todos los archivos y directorios creados por el paquete (archivo portátil)</value>
```

Comment:
"archivo portátil" means "portable file". Should it not be app or package instead of file?

Reply:
It should be "portable package"